src/Symfony/Component/Console/Descriptor/ApplicationDescription.php

```diff
@@ -134,15 +134,17 @@ private function inspectApplication()
     private function sortCommands(array $commands)
     {
         $namespacedCommands = array();
+        $globalCommands = array();
         foreach ($commands as $name => $command) {
             $key = $this->application->extractNamespace($name, 1);
             if (!$key) {
-                $key = '_global';
+                $globalCommands['_global'][$name] = $command;
```

Comment:
I'm not sure what you meant here. Anyway I know that `$globalCommands['_global']` feels redunant but in my opinion is it fine. The `array_merge` call later looks pretty straightforward this way.